src/control-plane/backend/lambda/api/store/dynamodb/dynamodb-store.ts

```diff
@@ -48,7 +48,7 @@ export class DynamoDbStore implements ClickStreamStore {
         region: project.region,
         environment: project.environment,
         pipelineId: '',
-        status: 'ACTIVED',
+        status: {},
```

Comment:
Will it be a change to break existing data?

Reply:
This is an initial value that does not affect existing data.

Reply:
How about using the `status` value with different types(`string` or `object`) when getting the project detail?

Reply:
The **pipeline.status** should be `PipelineStatus` object, so string is an error initial value.

Reply:
My mistake, I mistook it for **pipeline.status**, and **project.status**  is indeed a `string` type. And reset this attribute is `ACTIVATED`.